src/data/queries/tests/audienceTopic.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { queries } from '@/data/queries'
+import { ParagraphAudienceTopics } from '@/types/dataTypes/drupal/paragraph'
+import mockData from '@/mocks/audienceTopics.mock.json'
+
+// Adding this because next-drupal has some bad type definitions.
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+/* @ts-ignore */
+const audienceTopicsMocks: ParagraphAudienceTopics[] = mockData
+
+describe('paragraph--audience_topics formatData', () => {
```

Comment:
these are fairly repetitive. It might be possible to DRY this out if we're willing to do dynamic imports to get the mockData and type files in here. 

Also occurs to me that we should be testing that what is output is of proper type as well.